src/Microsoft.ML/Microsoft.ML.csproj

```diff
@@ -1,44 +1,24 @@
-<Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="$(RepoRoot)eng/pkg/Pack.props"/>
+<Project Sdk="Microsoft.Build.NoTargets">
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <!-- To include the DLL and PDBs of the underlying projects
-    We need to remove the IncludeBuildOutput=false. This also generates a
-    Microsoft.ML.dll in the nuget package, and a Microsoft.ML.pdb in the .snupkg
-    But these are to be ignored, as they don't contain anything meaningful-->
-    <!-- <IncludeBuildOutput>false</IncludeBuildOutput> -->
+    <NoTargetsDoNotReferenceOutputAssemblies>false</NoTargetsDoNotReferenceOutputAssemblies>
+    <DisableImplicitFrameworkReferences>false</DisableImplicitFrameworkReferences>
```

Comment:
Why are framework references required if this project doesn't compile anything? Nit: We try to not use netstandard2.0 as the TFM if not necessary for compilation. I would update this to NetCurrent or whatever property exists in this repo to indicate the latest supported .NETCoreApp TFM.

Reply:
For ApiCompat.  This project bundles many assemblies.  We need framework references to give APICompat the assembly closure.

Reply:
Makes sense. A code comment explaining that would help.